api/apps/api/test/project/update-project-calibration.e2e-spec.ts

```diff
@@ -32,3 +32,11 @@ test(`updating a project calibration with incorrect ranges should throw an excep
     .ThenShouldFailWhenStartingAnScenarioCalibrationWithA()
     .RangeWithNegativeNumbers();
 });
+
+test(`updating a project calibration range should not work if user is not in project`, async () => {
+  await fixtures.GivenProjectWasCreated();
+
+  const request = await fixtures.WhenProjectCalibrationIsUpdatedAsNotIncludedUser();
```

Comment:
consider splitting into:
```
const projectId = await fixtures.GivenProjectWasCreated()
const userToken = await fixtures.GivenUserNotInProject(projectId)
const result = await fixtures.WhenUpdatingCalibration(projectId, userToken)
fixtures.ThenActionIsDenied(result)
```

Reply:
I did so it is easier to understand the steps on the test.